docs/sources/mimir/operators-guide/architecture/components/ingester.md

```diff
@@ -5,11 +5,11 @@ description: "The ingester writes incoming series to long-term storage."
 weight: 30
 ---
 
-# Grafana Mimir ingester
+# Understanding the Grafana Mimir Ingester Component
```

Comment:
```suggestion
# Grafana Mimir ingester
```
The title must match the metadata, and the other components have parallel structure to the original. Reverting.